web/src/app/shared/models/links.ts

```diff
@@ -1,9 +1,5 @@
-export class Links {
-  constructor(
-    public self?: { 'href': string },
-    public distributionTargets?: { 'href': string },
-    public accessItems?: { 'href': string },
-    public allWorkbasketUrl?: { 'href': string },
-    public removeDistributionTargets?: {'href': string}
-  ) { }
+export interface Links {
```

Comment:
I am not sure if i like this change. With these computed Property names we do not know which fields this class provides. This at least needs a comment or a documentation link, that explains what links could be provided

Reply:
The issue here: Our backend is not consistent yet. Therefore the hateoas links provided by the backend are not consistent. Proposal: We live with it for now and specify the links whenever the backend is ready

Reply:
This sounds like a reasonable solution. Maybe we should comment this and create a corelating issue, to the issue for the [backend](https://taskana.atlassian.net/browse/TSK-1084)

Reply:
https://taskana.atlassian.net/browse/TSK-1294